src/testData.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { fromJS } from 'immutable';
```

Comment:
These are mocks. You can rename into `data.mock.ts` or something similar with `.mock.ts` postfix